spec/models/comment_spec.rb

```diff
@@ -67,6 +67,17 @@
 
   end
 
+  describe 'report_reasons' do
+
+    let(:comment) { FactoryGirl.build(:comment) }
+
+    it 'returns an array of strings' do
+      expect(comment.report_reasons).to be_an(Array)
+      expect(comment.report_reasons).to all(be_a(String))
```

Comment:
Actually, you're right. The content doesn't actually matter here (in that, there's no validation to say that it _needs_ to be one of those exact strings). I'd drop the expectation of it being an `Array`, because `.to all()` makes the same assertion.